File: LB-Assignments/Assignment_9/program1.c
#include<stdio.h>

int CountEven(int iNo)
{
    int iCnt = 0;
    int iRem = 0;

    while(iNo != 0)
    {
        iRem = iNo % 10;

        if(iRem %2 == 0)
        {
            iCnt++;
        }

        iNo = iNo/ 10;
    }

    return iCnt;

}
int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter Number : ");
    scanf("%d",&iValue);

    iRet = CountEven(iValue);

    printf("%d",iRet);
    
    return 0;
}

File: LB-Assignments/Assignment_20/program1.java
/*
program to find sum of all even numbers up to N
*/
import java.util.*;

class Logic
{
    int sumEvenNumbers(int iNo)
    {
        int iCnt = 0, iSum =0;

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            if(iCnt % 2 == 0)
            {
                iSum += iCnt;
            }
        }
        return iSum;
    }
}
class program1
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue =0, iRet = 0;

        System.out.println("Enter number : ");
        iValue = sobj.nextInt();

        Logic lobj = new Logic();

        iRet = lobj.sumEvenNumbers(iValue);

        System.out.println("sum of all even numbers is : "+iRet);

    }
}